packages/loll-component/index.js

```diff
@@ -53,7 +49,7 @@ export default function component (config) {
 
     for (let key in props) {
       if (props[key] !== comp.props[key]) {
-        comp.props = props
+        comp.props[key] = props[key]
```

Comment:
@estrattonbailey This is not related to ultradom, but looked like a bug so I did it.